build/images/Dockerfile.ubi

```diff
@@ -0,0 +1,9 @@
+FROM antrea/base-ubi:2.14.0
```

Comment:
Do we need both approaches / Dockerfiles (`build/images/Dockerfile.build.ubi` and `build/images/Dockerfile.ubi`) here as well, or can we just get away with `build/images/Dockerfile.build.ubi`?

Reply:
Done. Remove `build/images/Dockerfile.ubi`, just keep `build/images/Dockerfile.build.ubi`